pkg/argparser/args.go

```diff
@@ -18,7 +18,12 @@ type argMap struct {
 	flags []string
 }
 
+// SetArg sets a flag and value in the map
 func (a *argMap) SetArg(flag, arg string, isSpace bool) {
+	// if flag is empty, return
+	if len(flag) == 0 {
+		return
+	}
```

Comment:
Is this new if block needed? Is this tested in the updated test code? 🤔

Reply:
when flag is empty. It makes no sense. Added in unit test.